src/sentry/static/sentry/app/views/issueList/header.tsx

```diff
@@ -123,15 +120,18 @@ function IssueListHeader({
             <li key={tabQuery} className={query === tabQuery ? 'active' : ''}>
               <a onClick={() => onTabChange(tabQuery)}>
                 {queryName}{' '}
-                <StyledQueryCount
-                  count={queryCounts[tabQuery]}
-                  max={TAB_MAX_COUNT}
-                  tagType={
-                    (tabQuery === Query.NEEDS_REVIEW && 'warning') ||
-                    (tabQuery === Query.UNRESOLVED && 'default') ||
-                    undefined
-                  }
-                />
+                {queryCounts[tabQuery] && (
+                  <StyledQueryCount
+                    count={queryCounts[tabQuery].count}
+                    max={queryCounts[tabQuery].hasMore ? TAB_MAX_COUNT : 1000}
```

Comment:
can we receive 1000 as a prop instead of hard-coding it here, similar to how IssueListFilters and IssueListActions receives `queryMaxCount`

you can also do this in a follow-up PR since I know getting this out is time-sensitive

Reply:
I can do that